src/analyze/dependencies.ts

```diff
@@ -304,19 +304,27 @@ export async function runDependencyAnalysis(
     stats.dependencyCount.duplicate = duplicateDependencies.length;
 
     for (const duplicate of duplicateDependencies) {
-      const severityColor =
-        duplicate.severity === 'exact' ? colors.blue : colors.yellow;
-
-      let message = `${severityColor('[duplicate dependency]')} ${colors.bold(duplicate.name)} has ${duplicate.versions.length} installed versions:`;
+      // Disable colors completely when NO_COLOR is set (for consistent test snapshots)
+      const noColor = process.env.NO_COLOR === '1';
```

Comment:
why?

this feels like a workaround rather than a necessity.

are you doing this because we didn't have any snapshots yet which use colours? we should just set FORCE_COLOR to `1` in the vitest config env (although vitest 4 might already do this)